scripts/plotting/reprocessed_gatk4_samples_pca/project_reprocessed_gatk4_samples.py

```diff
@@ -0,0 +1,114 @@
+"""
+Perform PCA on the three 1kG samples,
+reprocessed using the GATK4 pipeline.
+"""
+
+import click
+import pandas as pd
+import hail as hl
+from hail.experimental import lgt_to_gt
+from hail.experimental import pc_project
+from bokeh.models import CategoricalColorMapper
+from bokeh.palettes import turbo  # pylint: disable=no-name-in-module
+from bokeh.io.export import get_screenshot_as_png
+
+HGDP1KG_TOBWGS = (
+    'gs://cpg-tob-wgs-analysis/1kg_hgdp_tobwgs_pca/'
+    'v1/hgdp1kg_tobwgs_joined_all_samples.mt'
+)
+
+LOADINGS = 'gs://cpg-tob-wgs-analysis/1kg_hgdp_nfe/v0/loadings.ht/'
+
+REPROCESSED_1KG = 'gs://cpg-tob-wgs-test/1kg_reprocessed/warp_gatk4/v0.mt'
+
+SCORES = 'gs://cpg-tob-wgs-analysis/1kg_hgdp_nfe/v0/scores.ht/'
+
+EIGENVALUES = 'gs://cpg-tob-wgs-analysis/1kg_hgdp_nfe/v0/eigenvalues.csv'
+
+
+@click.command()
+@click.option('--output', help='GCS output path', required=True)
+@click.option('--pop', help='Population to subset from the 1KG (e.g. afr, nfe)')
+def query(output, pop):  # pylint: disable=too-many-locals
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    mt = hl.read_matrix_table(HGDP1KG_TOBWGS)
+    if pop:
+        # Get samples from the specified population only
+        mt = mt.filter_cols(
+            (mt.hgdp_1kg_metadata.population_inference.pop == pop.lower())
+            | (mt.s.contains('TOB'))
+        )
+    else:
+        mt = mt.filter_cols(mt.s.contains('TOB'))
+
+    mt = mt.annotate_rows(af=hl.agg.mean(mt.GT.n_alt_alleles()) / 2)
+    loadings = hl.read_table(LOADINGS)
+    loadings = loadings.annotate(af=mt.rows()[loadings.key].af)
+    reprocessed_samples = hl.read_matrix_table(REPROCESSED_1KG)
+    reprocessed_samples = reprocessed_samples.annotate_entries(
+        GT=lgt_to_gt(reprocessed_samples.LGT, reprocessed_samples.LA)
+    )
+    ht = pc_project(reprocessed_samples.GT, loadings.loadings, loadings.af)
```

Comment:
Did you double-check that this produces reasonable results? For example, it might be good to verify that a previous sample (that was used for the PCA) gets the exact same coordinates when projecting it again.